all-spark-notebook/Dockerfile

```diff
@@ -16,9 +16,10 @@ RUN conda install boto3 \
     && pip install git+https://github.com/moj-analytical-services/etl_manager.git#egg=etl_manager \
     && pip install git+https://github.com/moj-analytical-services/dataengineeringutils.git \
     && pip install git+https://github.com/moj-analytical-services/gluejobutils/#egg=gluejobutils \
+    && pip install --upgrade nbstripout \
```

Comment:
again, as said in other comment, not sure what was the reason for the `--upgrade` but it shouldn't harm really ☺️